app/main/forms.py

```diff
@@ -0,0 +1,36 @@
+from flask_wtf import Form
+from wtforms import StringField, DateField
+from wtforms.validators import DataRequired, Optional, Regexp
+from datetime import datetime
+
+
+class ServiceUpdateAuditEventsForm(Form):
+    audit_date = DateField(
+        'Audit Date',
+        format='%Y-%m-%d',
+        validators=[Optional()])
+    acknowledged = StringField(
+        'acknowledged',
+        default="false",
+        validators=[
+            Regexp('^(all|true|false)$'),
```

Comment:
Had lots of issues with rendering the Radio Fields, so opted for toolkit HTML and String field to back it.